src/index.html

```diff
@@ -22,6 +22,366 @@
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h3 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h3 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
```

Comment:
The product cards are repetitive. Each card displays the same product details, which was previously noted as an issue. Consider updating each card with unique product details to showcase a diverse product catalog.